tests/Providers/TestCountriesServiceProvider.php

```diff
@@ -1,27 +1,30 @@
 <?php
 
+namespace PodPoint\Countries\Tests;
+
 use Illuminate\Config\Repository;
-use Mockery as m;
+use Illuminate\Contracts\Foundation\Application;
+use Mockery;
 use PodPoint\Countries\Providers\CountriesServiceProvider;
-use Stubs\ApplicationStub;
+use PHPUnit\Framework\TestCase as PHPUnitTestCase;
 
-class TestCountriesServiceProvider extends PHPUnit_Framework_TestCase
+class TestCountriesServiceProvider extends PHPUnitTestCase
 {
     /**
-     * Laravel application stub.
+     * Application mock.
      *
-     * @var ApplicationStub
+     * @var Mockery
      */
-    private $app;
+    private $appMock;
 
     /**
-     * Set up mocks.
+     * Set up tests.
      *
      * @return void
      */
     public function setUp()
     {
-        $this->app = new ApplicationStub();
+        $this->appMock = Mockery::mock(Application::class);
```

Comment:
Could use: 
```php
$this->appMock = $this->createMock(Application::class);
```

Which should theoretically do the same and means you can remove `mockery/mockery` from the `require-dev`

Reply:
Thanks, good shout - have switched to PHPUnit mocks now and removed the need for Mockery.